pyproject.toml

```diff
@@ -39,28 +39,29 @@ include = ["data/*.json"]
 #   - https://semver.org/
 # See here for "caret" style dependency specifications:
 #   - https://python-poetry.org/docs/dependency-specification/
-python = ">= 3.8, < 3.11"
-numpy = "^1.22"
-pandas = "^1.1"
-h5py = "^3.2"
-lxml = "^4.6"
-lasio = "^0.29"
-scipy = "^1.7.3"
-numba = ">=0.52, < 1.0"
-joblib = "^1.1.0"
+python = ">= 3.8.1, < 3.12"
```

Comment:
is the < 3.12 okay here, given the current numba limitation?

Reply:
Yep good catch, I've changed it back for now.